serializers/src/OBJ/objSerializer.ts

```diff
@@ -87,14 +93,16 @@ export class OBJExport {
                 for (i = 0; i < trunkUV.length; i += 2) {
                     output.push("vt " + trunkUV[i] + " " + trunkUV[i + 1]);
                 }
+                currentTextureV++;
```

Comment:
Shouldn't this line be inside the loop?

Reply:
I do not understand how my tests worked like this :-)